src/App.js

```diff
@@ -2,33 +2,11 @@ import { Routes, Route } from 'react-router-dom';
 // import { Container } from 'react-bootstrap';
 import Home from './components/home/Home';
 import Company from './components/item/Company';
-import './App.css';
```

Comment:
- [ ] Please, consider making your design similar as it is required with the original one (https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs))